tools/ui-components/src/button/button.tsx

```diff
@@ -59,7 +61,11 @@ const computeClassNames = ({
         'bg-default-background-info',
         'text-default-foreground-info',
         ...(disabled
-          ? []
+          ? [
+            'active:before:bg-default-background-quaternary',
+            'active:before:opacity-0',
+            'focus:ring-0'
```

Comment:
We should apply the same rule as https://github.com/freeCodeCamp/freeCodeCamp/pull/45739#discussion_r857155977 to this.

```suggestion
            'active:before:hidden'
```